src/Evolution/Systems/CurvedScalarWave/Worldtube/Triggers/InsideHorizon.hpp

```diff
@@ -0,0 +1,44 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <limits>
+#include <pup.h>
+
+#include "Evolution/Systems/CurvedScalarWave/Worldtube/Tags.hpp"
+#include "ParallelAlgorithms/EventsAndTriggers/Trigger.hpp"
+#include "Utilities/Serialization/CharmPupable.hpp"
+
+namespace Triggers {
+
+/*!
+ * \brief This trigger is true when the worldtube is entirely within a
+ * coordinate sphere of radius 1.99 M.
+ */
+class InsideHorizon : public Trigger {
+ public:
+  /// \cond
+  InsideHorizon() = default;
+  explicit InsideHorizon(CkMigrateMessage* /*unused*/) {}
+  using PUP::able::register_constructor;
+  WRAPPED_PUPable_decl_template(InsideHorizon);  // NOLINT
+  /// \endcond
+
+  static constexpr Options::String help =
+      "Triggers if the worldtube is entirely inside a coordinate sphere with "
+      "radius 1.99 M.";
```

Comment:
Add "centered on the origin in the inertial frame".

Also, should 1.99 be an option in the input file? Does this depend on the mass of the larger black hole always being 1M?

Reply:
This does assume that the central black hole has mass 1 but that is already assumed in a few places in the code. I am hesitant to make this an input file choice as only a very small range of radii make sense here and I found 1.99 M to be a very robust choice.